android/src/main/kotlin/co/sunnyapp/flutter_contact/Item.kt

```diff
@@ -59,6 +59,24 @@ class PhoneType : ItemType(otherType = Phone.TYPE_CUSTOM, labelField = Phone.LAB
     }
 }
 
+class EmailType : ItemType(otherType = Email.TYPE_CUSTOM, labelField = Email.LABEL, typeField = Email.TYPE) {
+    override fun calculateTypeInt(type: String?): Int {
+        return when (type?.toLowerCase()) {
+            "home" -> Email.TYPE_HOME
+            "work" -> Email.TYPE_WORK
+            "mobile" -> Email.TYPE_MOBILE
```

Comment:
do we need Email type mobile?

Reply:
I don't care all emails of any label I will fetch

Reply:
not sure why we need it in first place, email home,work have some sense 
mobile part just looks like copy from Phone.TYPE_ 
```
Phone.TYPE_HOME -> "home"
        Phone.TYPE_WORK -> "work"
        Phone.TYPE_MOBILE -> "mobile"
```